docs/guides/advanced-cluster-new-sharding-schema.md

```diff
@@ -406,29 +407,25 @@ resource "mongodbatlas_advanced_cluster" "test" {
   }]
   lifecycle { # avoids future non-empty plans as instance size start to scale from initial values
     ignore_changes = [ 
-      replication_specs[0].region_configs[0].electable_specs[0].instance_size, 
-      replication_specs[0].region_configs[0].analytics_specs[0].instance_size, 
-      replication_specs[1].region_configs[0].electable_specs[0].instance_size,
-      replication_specs[1].region_configs[0].analytics_specs[0].instance_size
+      replication_specs[0].region_configs[0].electable_specs.instance_size, 
+      replication_specs[0].region_configs[0].analytics_specs.instance_size, 
+      replication_specs[1].region_configs[0].electable_specs.instance_size,
+      replication_specs[1].region_configs[0].analytics_specs.instance_size
     ]
   }
 }
 ```
 
 While the example initially defines 2 symmetric shards, auto-scaling of `electable_specs` or `analytic_specs` can lead to asymmetric shards due to changes in `instance_size`.
 
--> **NOTE:** In the following scenarios, a `mongodbatlas_advanced_cluster` using the new sharding configuration (single `replication_specs` per shard) might not have shard-level auto-scaling enabled:
-1. Configuration was defined prior to version 1.23.0 when auto-scaling per shard feature was released.
-2. Cluster was imported from a legacy schema (For example, `mongodbatlas_cluster` or `mongodbatlas_advanced_cluster` using `num_shards` > 1).
-In these cases, you must update the cluster configuration to activate the auto-scaling per shard feature. This can be done by temporarily modifying a value like `compute_min_instance_size`.
+-> **NOTE:** In the following scenarios, a `mongodbatlas_advanced_cluster` using the new sharding configuration (single `replication_specs` per shard) might not have shard-level auto-scaling enabled: <br/>1. Configuration was defined prior to version 1.23.0 when auto-scaling per shard feature was released.<br/>2. Cluster was imported from a legacy schema (For example, `mongodbatlas_cluster` or `mongodbatlas_advanced_cluster` using `num_shards` > 1).<br/><br/>In these cases, you must update the cluster configuration to activate the auto-scaling per shard feature. This can be done by temporarily modifying a value like `compute_min_instance_size`.
```

Comment:
Would add the 3rd case of users transitioning from 1.x directly to 2.x, the transition does not trigger any update operation so autoscaling will remain CLUSTER wide.